cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -112,6 +112,54 @@ message UnregisterSourceSegyFileRequest {
 
 message UnregisterSourceSegyFileResponse {}
 
+message IngestSourceSegyFileRequest {
+    Identifier file = 1;                       // [required] The file to queue for ingestiong
+    /**
+        [optional] Selected step to start ingestion. Leave blank to start from last completed step.
+        [Accepted values]
+           1 -- insert binary and text headers
+           2 -- insert trace headers
+           3 -- insert trace data
+           4 -- compute coverage polygon and additional metadata (trace count, valid lines, etc)
+           6 -- insert trace offset indices
+        [Use cases]
+        * Usual ingestion:
+           If the last successful step was 0 (REGISTER), the file will be fully ingested regardless of start_step
+        * Completing a failed ingestion:
+           If the last successful step was 3 (INSERT_DATA), and no start step is selected, only coverage will be
+           computed. Previously ingested headers and traces will be preserved in this case.
+        * Forcing reingestion:
+           If start_step is 1 (INSERT_FILE_HEADERS), regardless of previous ingestion status, all headers
+           and data will be deleted and the whole file will be forcefully reingested
+        * Forcing a single step to be rerun:
+           If the file is ingested (last step is COMPUTE_COVERAGE), by using start_step = 4 (COMPUTE_COVERAGE),
+           for example, only coverage will be computed while headers and traces will be preserved
+    **/
+    FileStep start_step = 2;
```

Comment:
If we omit anything from the v0 API I think we should omit this. It had a purpose back when the python ingestion worker would copy each and every trace to Bigtable. It's much less useful these days.